site/includes/homepage.css

```diff
@@ -73,23 +88,50 @@ a:visited{
 #header a:visited{
   color:#4AA0C7;
 }
-#header ul{
+
+#header a span{
+  display: inline-block; 
+  margin-bottom: 5px;
+}
+
+@media (max-width: 640px){
+    #header a span{
+        display: none;
+    }
+}
+
+#header ul {
   position: relative;
   top: 0px;
   left: 0px;
-  min-height: 41px;
+  min-height: auto;
```

Comment:
This should probably just be height:auto. Before your change the intent of this code was to give the header a height of 41px, but allow it to get deeper if, say, the page was too narrow and some of the buttons wanted to newline. Using min-height would still allow it to expand downwards to accomodate height changes.

If you are handling height changes programatically now, this should probably be height:auto not min-height